images/makeproject/makeproject-step2.sh

```diff
@@ -90,6 +90,18 @@ yes | bin/update_versions
 #   --command_line "$((i * 4096)) $(((i + 1) * 4096))" \
 #   --wu_name "${wu_name}"
 #done
+# 2^33 seeds per boinc workunit * 2^15 boinc work units = 2^48 seeds total (PackCrack)
+#for i in {0..32767}; do
+# wu_name="packcrack_1.00_17179869184_$i"
```

Comment:
Can we change the name so its consistent with the new work unit size (8589934592)?